client/src/globals/UserContext.js

```diff
@@ -23,16 +39,35 @@ export const UserContextProvider = (props) => {
                 else return res.json()
             })
             .then(res => {
-                if (status === 200 && res.success === true) return setUserId(res.userId);
+                if (status === 200 && res.success === true) {
+                    setIsComplete(true)
+                    setIsSubscribed(res.isSubscribed)
+                    setUserEmail(res.userEmail)
+                    return setUserId(res.userId);
+                }
                 else throw Error("error");
```

Comment:
Since the `if condition` returned, there is no need to use `else` under that, but it's still fine.